code/game/objects/items/devices/player.dm

```diff
@@ -4,16 +4,6 @@
 	var/current_track = ""
 	var/obj/item/holder = null
 	var/list/songs = list(
```

Comment:
Does this means the item is useless now ?
If yes it should be deleted or made unobtainable

Reply:
everything else looks fine tho

Reply:
Will address.